core/feconf.py

```diff
@@ -499,6 +499,8 @@ def get_empty_ratings() -> Dict[str, int]:
 DATAFLOW_TEMP_LOCATION = 'gs://todo/todo'
 DATAFLOW_STAGING_LOCATION = 'gs://todo/todo'
 
+OPPIA_PYTHON_PACKAGE_PATH = './build/oppia-beam-job-0.0.1.tar.gz'
```

Comment:
See https://github.com/oppia/oppia/pull/13938#discussion_r719809576

I'm also confused now (looking at the name) about whether this is versioned or not versioned.

Reply:
Added a comment.